src/Drupal/EntityFixtureTrait.php

```
@@ -0,0 +1,126 @@
+        continue;
+      }
+
+      if (is_array($value)) {
+        $value[0] = $fixture_path . $basename;
+        $stub->setValue($name, $value);
+      }
+      else {
+        $stub->setValue($name, $fixture_path . $basename);
+      }
+    }
+  }
```

PHPMD (2.15.0)

[warning] The method entityFixtureExpand() has a Cyclomatic Complexity of 18. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)